plugins/azure-devops-backend/api-report.md

```diff
@@ -20,11 +20,10 @@ import { RepoBuild } from '@backstage/plugin-azure-devops-common';
 import { Team } from '@backstage/plugin-azure-devops-common';
 import { TeamMember } from '@backstage/plugin-azure-devops-common';
 import { UrlReader } from '@backstage/backend-common';
-import { WebApi } from 'azure-devops-node-api';
 
 // @public (undocumented)
 export class AzureDevOpsApi {
-  constructor(logger: Logger, webApi: WebApi, urlReader: UrlReader);
+  constructor(logger: Logger, urlReader: UrlReader, config: Config);
```

Comment:
This is a breaking change, if we want to make it I suggest that we instead make the constructor private and create a new `create()` method that accepts an options object and internally calls the private constructor. That make the API a bit more flexible and less prune to breaking changes in the future.